pkg/infra/pulumi_aws/iac/static_s3_website.ts

```diff
@@ -9,21 +9,24 @@ export const createStaticS3Website = (
     staticUnit: string,
     indexDocument: string,
     contentDeliveryNetworkId: string,
-    lib: CloudCCLib
+    lib: CloudCCLib,
+    appName: string
 ) => {
     // Create an S3 bucket
 
-    const bucketArgs: aws.s3.BucketArgs = {}
-
-    if (indexDocument != '' && contentDeliveryNetworkId == '') {
-        bucketArgs['website'] = {
-            indexDocument: indexDocument,
+    lib.account.accountId.apply((accountId) => {
```

Comment:
nit: I think its cleaner to just use the apply to get the string for the bucket name since thats all that we use it for and not have all the other code wrapped in the apply. Dont think it matters for functionality though